### PR TITLE
Define Design System Colors

### PR DESCRIPTION
**Pull request by brad-decker**
### Changes
1. Adds a new variables folder at `ui/app/css/variables`
2. Adds a new `colors.scss` file in said folder
3. Moves valid color declarations from old variables file into new colors file
4. Supplements the previously existing colors to create a full list of defined colors from the design system.
5. Updates the import order to give precedence to the new setup
6. Removes superfluous color declarations that are not used in the code or were duplicated.

### CODE REVIEW

File: ui/app/css/variables/colors.scss [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+$Orange-000: #fef5ef;
+$Orange-100: #fde2cf;
+$Orange-200: #fbc49d;
+$Orange-300: #faa66c;
+$Orange-400: #f8883b;
+$Orange-500: #f66a0a;
+$Orange-600: #c65507;
+$Orange-700: #954005;
+$Orange-800: #954005;
```
Comment (Gudahtt) [inline ui/app/css/variables/colors.scss:56]: This was a dupe of `$Orange-700` - I've updated it to use the color referenced in the Figma:
```suggestion
$Orange-800: #632b04;
```

File: ui/app/css/variables/colors.scss [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+$Orange-900: #321602;
+
+$Yellow-100: #fefcde;
+$Yellow-500: #ffd33d;
```
Comment (Gudahtt) [inline ui/app/css/variables/colors.scss:60]: The Figma uses a different color here:
```suggestion
$Yellow-500: #f66a0a;
```

Comment (Gudahtt) [reply ui/app/css/variables/colors.scss:60]: Though the `Yellow` referenced in the figma is `#ffd33d` :thinking: 
I guess yours is correct here as well.

File: ui/app/css/variables/colors.scss [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+$Grey-300: #9fa6ae;
+$Grey-400: #848c96;
+$Grey-500: #6a737d;
+$Grey-600: #535a61;
+$Grey-700: #3b4046;
+$Grey-800: #24272a;
+$Grey-900: #141618;
+
+$Green-000: #f3fcf5;
+$Green-100: #d6ffdf;
```
[the remark this answers is not in the document]
Comment (Gudahtt) [reply ui/app/css/variables/colors.scss:27]: Yours matches the color used in Figma, but not the written hex code. So I guess yours is correct.

**Review by Gudahtt — APPROVED** [review]
LGTM!

File: ui/app/css/variables/colors.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (brad-decker) [reply ui/app/css/variables/colors.scss:27]: Yeah there were a couple instances of that. We should probably move away from writing the hex value directly on the sheet and instead just use the color variable name. We can figure out the hex values by clicking the swatch

**Comment by brad-decker** [comment]
[Update fonts](https://trello.com/c/giqTMDWN/59-update-fonts)

**Comment by brad-decker** [comment]
[Update fonts](https://trello.com/c/giqTMDWN/59-update-fonts)

**Comment by brad-decker** [comment]
[Update fonts](https://trello.com/c/giqTMDWN/59-update-fonts)

**Pull request merged**